js/defaults_dialog.js

```diff
@@ -20,185 +20,226 @@ helper.defaultsDialog = (function () {
         "settings": [
             {
                 key: "model_preview_type",
-                value: 3
+                value: 3, 
+                allow_reset: true
```

Comment:
We should not add this here. Rather create a separate list of settings that should not be changed during reset